.github/workflows/analysis_workflow.yml

```diff
@@ -0,0 +1,197 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.#
```

Comment:
Is it needed to have a # at the end of the line?

Reply:
Fixed.